features/yield-calculator/YieldCalculator.tsx

```diff
@@ -0,0 +1,120 @@
+import { Box, TextField, Button, Typography } from "@material-ui/core";
+import { FormEvent, useState } from "react";
+import styled from "styled-components";
+import EmpState from "../../containers/EmpState";
+
+const Container = styled.div`
+  padding: 1rem;
+  border: 1px solid #434343;
+`;
+
+const FormInput = styled.div`
+  margin-top: 20px;
+  margin-bottom: 20px;
+`;
+
+const MS_TO_S = 1000;
+const S_TO_DAYS = 60 * 60 * 24;
+const DAYS_TO_YEAR = 365;
+
+const YieldCalculator = () => {
+  const { empState } = EmpState.useContainer();
+  const [tokenPrice, setTokenPrice] = useState<number | null>(0.9575);
```

Comment:
nit: where did this magic number come from? Might be preferable to make it a named constant so it's clear that you're using it as the default input price.

Reply:
Its just a random default number that is a placeholder for the live market price of yUSD which we can grab from balancer when the time comes. I'll rename.

Reply:
```
// TODO: If the user has not selected an EMP (or has not even connected their web3 provider yet),
// then the EMP's expirationTimestamp will clearly be unavailable. In order to support explorability,
// this is a placeholder `daysToExpiry` that will be the default value for the textfield if no EMP is selected yet.
const PLACEHOLDER_DAYS_TO_EXPIRY = 30;
```

Reply:
```
// TODO: When the yUSD market goes live on an exchange, we can replace this
// `PLACEHOLDER_PRICE` with the real-time mid-market price for yUSD.
const PLACEHOLDER_PRICE = 0.9875;
```